prisma/schema.prisma

```diff
@@ -0,0 +1,24 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
```

Comment:
Need a good way to test functionality locally without setting up database. Could you update the README for that?

Reply:
@Kitenite The database is very much needed to test everything since chats and projects are now stored there. You can use the preview URL to see it with the database we've set up (https://vercel.live/open-feedback/gpt-react-designer-git-feature-mag-2589-magicul.vercel.app?via=pr-comment-visit-preview-link&passThrough=1)